src/Microsoft.Data/BaseColumn.cs

```diff
@@ -34,7 +34,11 @@ protected set
             }
         }
 
-        public long NullCount { get; protected set; }
+        public virtual long NullCount
```

Comment:
Why not `abstract`?  I guess the same could be said for GetValue and SetValue below.  (Should have said that last time)